js/models/chat/ChatMessage.js

```diff
@@ -50,9 +51,16 @@ export function processMessage(message) {
           const w = word.trim();
           if (wordsToAnchorify.includes(w)) return;
 
+          // Certain words are valid multihashes (e.g. that)
+          // const isGuid = isMultihash(w) && w.length > 15;
+          const isGuid = isMultihash(w) && w.length > 15;
+
           if ((w.startsWith('@') && w.length > 1) ||
             (w.startsWith('ob://') && w.length > 5) ||
-            isMultihash(w)) {
+            (w.startsWith('http://') && w.length > 7) ||
```

Comment:
Since the shortest url is the prefix plus one character plus a dot plus 2 characters (like http://a.io), should the length checks be 11, 12, and 8 instead?

Reply:
If that is indeed the shortest url, then yes.